ui/src/app/views/app/component.html

```diff
@@ -4,12 +4,20 @@
   <img class="portal-logo" src="assets/images/portal-logo.svg" alt="Researcher Portal"
     style="margin-left: 0.5rem; width: 13rem;">
 </div>
-<nav style="margin-top: 1rem;">
+<nav *ngIf="isSignedIn | async" style="margin-top: 1rem;">
   <a routerLink="/login" routerLinkActive="active">Switch User</a>
   <a routerLink="/repository" routerLinkActive="active">Select CDR</a>
 </nav>
 <!--
   Angular's builtin Router module will attach specific views here
   based on clicking routerLink elements (above) or router.navigate calls.
 -->
-<router-outlet></router-outlet>
+<div *ngIf="(user | async).isSignedIn">
```

Comment:
I can do this in on place, with if/else (done). There is also "if" (with no else) logic in other places of the template (i.e. the "stuff that shows up when logged in" is distributed over the page).